demosplan/DemosPlanCoreBundle/Logic/Statement/StatementDeleter.php

```diff
@@ -73,6 +79,7 @@ public function deleteOriginalStatementAttachmentByStatementId(string $statement
      * @throws UserNotFoundException
      * @throws ORMException
      * @throws Exception
+     * @throws \Exception
```

Comment:
duplicate throws

Reply:
done in [e753db9](https://github.com/demos-europe/demosplan-core/pull/3014/commits/e753db99a1aecc1a42c9037150bb18ae276363d7)